File: Ejercicios en clases/poo php/Three php/library/Descuentos.php
<?php
// Clase Descuentos que calcula los descuentos de salud, pensión y ARL
class Descuentos {
    public $salud;
    public $pension;
    public $arl;

    public function __construct($salud, $pension, $arl) {
        $this->salud = $salud;
        $this->pension = $pension;
        $this->arl = $arl;
    }

    // Método que calcula el total de descuentos
    public function totalDescuentos() {
        return $this->salud + $this->pension + $this->arl;
    }
}
?>

File: Ejercicios en clases/poo java/Three java/Library/Descuentos.java
public class Descuentos {
    private double salud;
    private double pension;
    private double arl;

    public Descuentos(double salud, double pension, double arl) {
        this.salud = salud;
        this.pension = pension;
        this.arl = arl;
    }

    public double getSalud() {
        return salud;
    }

    public double getPension() {
        return pension;
    }

    public double getArl() {
        return arl;
    }

    public double totalDescuentos() {
        return salud + pension + arl;
    }
}